src/priyanka_Kamble/Exam1/Exam.java

```diff
@@ -0,0 +1,24 @@
+package priyanka_Kamble.Exam1;
+
+public class Exam {
+	
+	int target = 9;
+
+	void getindex(int[] arr) {
+		for (int index = 0; index < arr.length; index++) {
+			for (int innerIndex = 1; innerIndex < arr.length; innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files